content/docs/user-guide/gems/reference/audio/wwise/audio-engine-wwise.md

```diff
@@ -25,6 +25,12 @@ In general you can use more recent versions of Wwise than specified above, but s
 
 1. (Recommended) Set the CMake cache variable `LY_WWISE_INSTALL_PATH` to the path where you installed Wwise. You can use `cmake-gui` to set this variable, or you can set it when you run the `cmake` build configuration command. By using this cache variable, CMake project regeneration will automatically be triggered if the variable is updated in the future.
 
+    You can use the following CMake build configuration command to set `LY_WWISE_INSTALL_PATH` from your source engine or project directory.
+
+    ```cmd
+    cmake configure -S . -B /build/<platform> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<o3de-packages> -DLY_WWISE_INSTALL_PATH=<wwise-installation>

```

Comment:
No leading slash for the build directory. And we have been leaving off the -S for the source directory in our examples, since we are running the command from the source directory.
```suggestion
    cmake configure -B build/<platform> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<o3de-packages> -DLY_WWISE_INSTALL_PATH=<wwise-installation>

```